lib/ngx/process.md

```diff
@@ -32,7 +32,7 @@ The API is still in flux and may change in the future without notice.
 Synopsis
 ========
 
-Enables privileged agent process and get process type
+Enables privileged agent process, get process type and get master process pid
 -----------------------------------------------------
```

Comment:
Hmm, this section name is too long. Maybe we should make it normal text instead of a title.